tools/theme/src/Theme.ts

```diff
@@ -476,11 +440,6 @@ function checkForIssues(
             }
         };
 
-        if (hasThemeAttribute) {
```

Comment:
Should we keep till all the downstream consumers has migrated to system? _Just a thought_

Reply:
I'd be for taking advantage of the opportunity and clean it altogether. We are not doing this for any other deprecation removal in our system. Let me know what you think.